ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -81,7 +81,7 @@ interface RegistrationTask {
   key: string;
   completionPropsKey: string;
   loadingPropsKey?: string;
-  title: string;
+  title: React.ReactNode;
```

Comment:
Used `React.ReactNode` for the titles so they can be either a string or an element.